sdk/data/aztables/CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
 * Convert `Start` and `Expiry` times in `AccessPolicy` to UTC format as required by the service.
 * Fixed `moduleName` to report the module name as part of telemetry.
 
+### Other Changes
+* Fixed bugs in some live tests.
```

Comment:
Does this impact users?

Reply:
Not really.  These errors aren't listed as part of the [table service errors[(https://docs.microsoft.com/en-us/rest/api/storageservices/table-service-error-codes).  I suspect that the test hadn't been run after it was updated.